src/org/usfirst/frc/team467/robot/Autonomous/Actions.java

```diff
@@ -219,15 +198,8 @@ public static ActionGroup testGrab() {
 		mode.addAction(elevatorToSwitch());
 		return mode;
 	}
-	
-	//SWITCH
-	
-	
+
 	//SWITCH - BASIC
-	/*
-	 * 
-	 * 
-	 */
```

Comment:
How about adding a proper java doc comment for these instead of custom comment styles.